annotation-metadata/annotation-metadata-version-1.1

```diff
@@ -0,0 +1,237 @@
+@startuml
+skinparam backgroundColor transparent
+skinparam classBackgroundColor LightGoldenRodYellow
+skinparam classarrows red
+skinparam classFontName Menlo
+skinparam colors yellow
+skinparam ActorBorderColor red
+skinparam classborderColor darkred
+left to right direction
+
+
+
+title Standardization Effort Version 1.1
+/'
+The axon is connected to the Presynaptic 
+sending a chemical to be released into the 
+Postsynaptic.
+'/
+Cell ---[#darkred]-* Soma
+Soma -[#darkred]-* Spine
+Soma ------[#darkred]-* Axon
+Soma -[#darkred]-* Dendrite
+Soma -[#darkred]-* Cilia 
+Dendrite -[#darkred]-* Shaft
+Dendrite -[#darkred]-* Spine
+Axon -[#darkred]-* Axon_Initial_Segment
+Axon -[#darkred]-* Bouton
+Gap_Junctions -[#darkred]-* Axon
+Myelin -[#darkred]-* Axon
+Synapse -[#darkred]-* Axon
+Synapse -[#darkred]-* Dendrite
+Synapse -[#darkred]-* Axon_Initial_Segment
+Synapse -[#darkred]-* Bouton
+Synapse ---[#darkred]-* Spine
+Synapse ---[#darkred]-* Shaft
+Synapse -[#darkred]-* Soma 
+Spine_Apparatus -[#darkred]* Spine
+Organelles -[#darkred]-* Gap_Junctions
+Organelles -[#darkred]-* Nucleus
+Organelles -[#darkred]-* Mitochondria
+Organelles -[#darkred]-* SynapticVesicle
+Organelles -[#darkred]-* Myelin
+Organelles -[#darkred]-* Cleft
+Organelles -[#darkred]-* Spine_Apparatus
+Nucleus -[#darkred]-* Soma
+Cleft -[#darkred]-* Synapse 
+Mitochondria -[#darkred]-* All_Cell_Compartments
+SynapticVesicle -[#darkred]-* Synapse
+
+class Cell{
+	*Cell Types:
+	Neuron : boolean, 
+	Glia : boolean
+	
+	*Neuron Types:
+	sensory_neurons: enum,
+	motor_neurons : enum,
+	interneurons: enum
+
+	*Glia Types: 
+		astrocytes : enum,
+		microglia : enum,
+		oligodendrocyte : enum
+
+	*Regions of Interest:
+	layer : enum,
+	brain_regions : enum,
+	cylinder : enum
+
+	*Optional Cell Properties:
+		multipolar : enum,
+		bipolar : enum,
+		unipolar : enum,
+		anaxonic : enum,
+		number_of_blood_vessel_cells : enum,
```

Comment:
What does a `number_of_blood_vessel_cells` attribute imply for a neuron? i.e., is this a property of a single cell, or is this something about a cell's surroundings?

Reply:
I was under the impression it references the cell's surroundings! It may make more sense to relocate it or alter the attribute name/data type to better fit, maybe `blood_vessel_cells : boolean` to eliminate the numerical aspect because I know that could be hard to account for.

This also may not be applicable, but I am also under the impression something along the lines of a RBC or WBC count could help depict abnormalities in data, leading us to want to keep this spot to enter a number/specify what the blood vessel is.